pkg/apis/certificates/v1beta1/defaults.go

```diff
@@ -32,6 +32,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 
```

Comment:
v1beta1 is no longer served in 1.22+, so I don't think we need to change defaults for it (or should). I would recommend reverting the v1beta1 changes

Reply:
updated.